server/src/auth/service.js

```diff
@@ -0,0 +1,33 @@
+const UserModel = require('../user/model')
+const bcrypt = require('bcryptjs')
+const jwt = require('jwt-simple')
+const cfg = require('../../config/auth')
+
+exports.login = (req, res) => {
+  // check if the body contains an email and a password
+  if (req.body.email && req.body.password) {
+    // body assignments
+    let email = req.body.email
+    let password = req.body.password
+    // find user by email
+    UserModel.findOne({ 'email': email })
+      .then(result => {
+        // compare hashes
+        bcrypt.compare(result.password, bcrypt.hashSync(password)).then((resu) => {
```

Comment:
Please separate the `then` to a new line and change the name of the variable,(optional: as well the parenthesis are not needed when you have only one argument)